extensions/wikia/FlowTracking/scripts/createPageTracking.js

```diff
@@ -15,20 +15,45 @@ define(
 				tracked = qs.getVal('tracked', false);
 
 			// Do not track if the step was tracked already or article exists
-			if (tracked || !isNewArticle() || !(isMainNamespace() || isAllowedSpecialPage())) {
+			if (tracked || !isNewArticle() || !isMainNamespace()) {
 				return;
 			}
 
 			if (flowParam || document.referrer) {
 				flowTrack.trackFlowStep(flowParam, {editor: editor});
-			} else if (namespaceId === 0) {
+			} else {
 				flowTrack.beginFlow(window.wgFlowTrackingFlows.CREATE_PAGE_DIRECT_URL, {editor: editor});
 				qs.setVal('flow', window.wgFlowTrackingFlows.CREATE_PAGE_DIRECT_URL);
-			} else if (namespaceId === -1) {
-				// TODO: direct-url to Special:CreatePage (WW-351)
 			}
 
-			// set 'tracked' query param to prevent tracking the same event when page is reloaded
+			setTrackedQueryParam(qs);
+		}
+
+		function trackOnSpecialCreatePageLoad(editor) {
```

Comment:
This method looks pretty similar to direct tracking method. Could we unify them somehow?